src/handlers/check_commits.rs

```diff
@@ -350,3 +367,139 @@ r#":warning: **Warning** :warning:
      - 123456789"#
     );
 }
+
+#[cfg(test)]
+fn make_opened_issues_event() -> IssuesEvent {
```

Comment:
```suggestion
fn make_opened_pr_event() -> IssuesEvent {
```
Would be a bit easier to understand, because most tests actually test the PR stuff, and it was weird seeing a function called "create issue" called, and then we modified it to actually create an issue (in `test_not_pr`) :)

Reply:
`issues_event` actually referenced the name of the struct `IssuesEvent`, but I can see the confusion, rename it to `make_opened_pr_event` as suggested.